governator-core/src/main/java/com/netflix/governator/visitors/ModuleSourceTracingVisitor.java

```diff
@@ -0,0 +1,26 @@
+package com.netflix.governator.visitors;
+
+import com.google.inject.spi.DefaultElementVisitor;
+import com.google.inject.spi.Element;
+import com.google.inject.spi.ElementSource;
+
+/**
+ * Visitor for logging the 'path' through which each binding was created
+ */
+public class ModuleSourceTracingVisitor  extends DefaultElementVisitor<String> { 
+    @Override 
+    protected String visitOther(Element element) {
+        Object source = element.getSource();
+        ElementSource elementSource = null;
+        while (source != null && source instanceof ElementSource) {
```

Comment:
I think you should be able to call instanceof on a null (which will fail the instanceof)